user_templates/variables.cfg

```diff
@@ -84,6 +84,10 @@ variable_max_bed_xy: 9999, 9999
 ## Minimum safe Z height to attach/detach probe
 variable_probe_min_z_travel: 20
 
+## If probe dock has a retainer and how far to drop to before detaching 
+variable_probe_dock_separate_retainer: False
+variable_probe_dock_retainer_location_z: 20
```

Comment:
I'm not sure if this is possible (depending of how the probe is working), but it look like that setting a value of 0 in the z value will just do nothing more than before. So these two variables may be merged into a single one that is 0 by default and you can add some value if needed.

Also just calling it "z_hop_after_probe_drop" or something like that could be a good idea to help people understand its use. But I'm unsure for now since I first need to understand how your probe is working ^^